docs/README.md

```diff
@@ -15,15 +15,25 @@ Grafana periodically updates the Docker image, [`docs-base`](https://hub.docker.
 First, make sure the Docker daemon is running on your machine. Then, follow these steps:
 
 1. On the command line, first change to the docs folder: `cd docs`.
-1. Run `make docs`. This launches a preview of the website with the current grafana docs at `http://localhost:3002/docs/grafana/latest/` which will refresh automatically when changes are made to content in the `sources` directory.
+2. Run `make docs`. This launches a preview of the website with the current grafana docs at `http://localhost:3002/docs/grafana/latest/` which will refresh automatically when changes are made to content in the `sources` directory.
 
 If you have the grafana/website repo checked out in the same directory as the grafana repo, then you can run `make docs-local-static` to use local assets (such as images).
 
 ---
 
 ## Content guidelines
 
-Edit content in the `sources` directory.
+Generally, one can edit content in the `sources` directory.
+
+NOTE: the following paths are built instead from a typescript file and are auto-generated. Please do not edit these files directly.
+Instead, navigate to the appropriate typescript source file and edit the content there, then follow the build instructions to generate the markdown files.
+
+- Transformations
+  - Auto-generated markdown location:
+    - docs/sources/panels-visualizations/query-transform-data/transform-data/index.md
+  - Typescript location for editing and instructions:
+    - scripts/docs/generate-transformations.ts
+    - public/app/features/transformers/docs/content.ts
```

Comment:
I'm not sure of the best way to word this, but just something to get across what kind of content is in each file.
```suggestion
    - scripts/docs/generate-transformations.ts - Includes all content not specific to a transformation.
    - public/app/features/transformers/docs/content.ts - Transformation-specific content.
```